ImageViewer/Source/Extensions/UIViewController.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 public extension UIViewController {
 
-    public func presentImageGallery(_ gallery: GalleryViewController, completion: (() -> Void)? = {}) {
+    func presentImageGallery(_ gallery: GalleryViewController, completion: (() -> Void)? = {}) {
```

Comment:
why remove the `public` access modifier?

Reply:
The Swift 5 compiler complained about it being redundant because it's already in a public extension.